src/main/io/serial.h

```diff
@@ -71,7 +71,8 @@ typedef enum {
     BAUD_1000000,
     BAUD_1500000,
     BAUD_2000000,
-    BAUD_2470000
+    BAUD_2470000,
+    BAUD_COUNT
```

Comment:
As this will need a documentation update, what exactly is `BAUD_COUNT` representing?

Reply:
it's the number of suppported baud rate settings.